<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cifrar</class>
 <widget class="QDialog" name="Cifrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="archivoDestinoL">
        <property name="text">
         <string>Nombre del archivo destino</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="archivoDestinoLE"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="contrasenaCB">
        <property name="text">
         <string>Utilizar contraseña personalizada</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="contrasenaL">
        <property name="text">
         <string>Contraseña personalizada</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="contrasenaLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="cancelarPB">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cifrarPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cifrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelarPB</sender>
   <signal>clicked()</signal>
   <receiver>Cifrar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
